common/src/edgeai_demo.cpp

```diff
@@ -0,0 +1,1147 @@
+                {
+                    multi_scaler_obj->multiScalerObj1.num_channels = m_cameraObj->sensorObj.num_cameras_enabled;
+                    multi_scaler_obj->multiScalerObj2.num_channels = m_cameraObj->sensorObj.num_cameras_enabled;
+                }
+
+                status = tiovx_multi_scaler_module_init(m_ovxGraph->context,
+                                                        &multi_scaler_obj->multiScalerObj1);
+                if (multi_scaler_obj->useSecondaryMsc)
+                {
+                    status = tiovx_multi_scaler_module_init(m_ovxGraph->context,
+                                                            &multi_scaler_obj->multiScalerObj2);
+                }
+
+                m_multiScalerObjs.push_back(multi_scaler_obj);
+
+                /**
+                 * This vector will contain index of all multiScalerObj which
+                 * will take input from the camera nodes.
+                 */
+                if(input->m_srcType == "camera")
+                {
+                    m_camMscIdxMap.push_back(m_multiScalerObjs.size()-1);
+                }
+            }
+
+            /**
+             * This map contains mosaic info (posX,posY,width,height...) about
+             * all inputs that will end up being a part of single mosaic that
+             * will eventually go to this particulat output.
+             */
+            commonMosaicInfo[output->m_instId].push_back(flow->m_mosaicInfoVec[i]);
+
+            /**
+             * Push all model names for particular output.
+             * Used to render text above mosaic windows.
+             */
+            modelName = model->m_modelPath;
+            if (modelName.back() == '/')
+            {
+                modelName.pop_back();
+            }
+            modelName = std::filesystem::path(modelName).filename();
+            commonMosaicTitle[output->m_instId].push_back(modelName);
+
+            /**
+             * This map contains index of all post-process nodes that will
+             * be the inputs to a single mosaic. This mosaic will eventually
+             * go to this particular output.
+             */
+            postProcMosaicConn[output->m_instId].push_back(m_postProcObjs.size() - 1);
+
+            if(output->m_sinkType == "display" && m_displayObj == NULL)
+            {
+                /* Init display module. */
+                m_displayObj = new TIOVXDisplayModuleObj;
+                tiovx_display_module_params_init(m_displayObj);
+
+                m_displayObj->input_width = output->m_width;
+                m_displayObj->input_height = output->m_height;
+                m_displayObj->input_color_format = VX_DF_IMAGE_NV12;
+
+                m_displayObj->params.outWidth = output->m_width;
+                m_displayObj->params.outHeight = output->m_height;
+
+                /* Rtos display has fixed resolution of 1920x1080.*/
+                m_displayObj->params.posX = (1920-output->m_width)/2;
```

Comment:
@s-jain1 Should we parameterize the resolution with some # defines or is this never going to change?  We could probably leverage those # defines elsewhere throughout this app

Reply:
Created a macro for the same